Tools/Scripts/build-and-analyze

```diff
@@ -80,6 +80,8 @@ def make_analyzer_flags(output_dir, args):
 
     if args.only_smart_pointers:
         additional_checkers.extend([
+            'webkit.NoUncountedMemberChecker',
+            'webkit.RefCntblBaseVirtualDtor',
             'alpha.webkit.UncountedCallArgsChecker',
             'alpha.webkit.UncountedLocalVarsChecker',
```

Comment:
Super nit:  I'd add keep these in alphabetical order, but it's okay to land without doing that.

Reply:
ok, I can report them.